Sources/Tracking/Util/DeviceInfo.swift

```diff
@@ -66,8 +66,22 @@ public class CIODeviceInfo: DeviceInfo {
         SdkVersion.version
     }
 
+    // Requirements:
+    // 1. Use - instead of _
+    // 2. We prefer to get the OS language that the user has set. First try to return that. If that does not succeed
+    // we default to the language set for the host app. The language returned for that will only return languagues
+    // that the app supports. If OS set to "es" but app does not support Spanish, then "es" will not be returned.
```

Comment:
`If OS set to "es" but app does not support Spanish, then "es" will not be returned.`

comment seems inconsistent with the code to me as looks like we will always prefer to return the os lang.

Reply:
`if let osSetLanguage = Locale.preferredLanguages.first` has the possibility of returning `null` and the `if` statement being `false`. That's when we will *not* be returning the OS lang.